default.nix

```diff
@@ -514,19 +411,20 @@ in let this = rec {
           hackGet
           extendHaskellPackages
           foreignLibSmuggleHeaders
-          stage2Script
           ghc
           ghcHEAD
-          ghc8_2_1
-          ghc8_0_1
+          ghc8_4_3
+          ghc8_2_2
+          ghc8_0_2
           ghc7
-          ghc7_8
           ghcIosSimulator64
-          ghcIosArm64
-          ghcIosArmv7
-          ghcAndroidArm64
-          ghcAndroidArmv7a
```

Comment:
If we're keeping the old symbols in the let-binding as deprecations, they still need to be exported.